src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const listItem = document.createElement('li');
+
+    listItem.textContent = key;
+    list.append(listItem);
+
+    if (Object.keys(data[key]).length !== 0) {
```

Comment:
в принципі можна не робити перевірку на 0
Object.keys(data[key]).length й так конвертується в true, якщо його значення більше 0

Reply:
Погоджуюся з Андрієм.
```suggestion
    if (Object.keys(data[key]).length) {
```